cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel.cpp

```diff
@@ -17,7 +17,10 @@
 #include <iostream>
 #include "oneapi/dal/algo/jaccard/common.hpp"
 #include "oneapi/dal/algo/jaccard/vertex_similarity_types.hpp"
-#include "oneapi/dal/data/graph.hpp"
+#include "oneapi/dal/data/graph_service_functions.hpp"
+
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
```

Comment:
re-define api for  intersection, size_t, name, nodeid, pointers, consts etc

Reply:
@andreynv 
All content of kernel (except API) will be changed by Alexey. Currently, here is some kind of checks that API works.